src/core/thread/mle_router.cpp

```diff
@@ -534,11 +534,27 @@ void MleRouter::ResetAdvertiseInterval(void)
     return;
 }
 
-void MleRouter::SendAdvertisement(void)
+void MleRouter::SendMulticastAdvertisement(void)
 {
-    Error        error = kErrorNone;
     Ip6::Address destination;
-    TxMessage   *message = nullptr;
+
+    destination.SetToLinkLocalAllNodesMulticast();
+    SendAdvertisement(destination);
+}
+
+void MleRouter::ScheduleUnicastAvdertisementTo(const Router &aRouter)
```

Comment:
```suggestion
void MleRouter::ScheduleUnicastAdvertisementTo(const Router &aRouter)
```

Reply:
Thanks, @EskoDijk, for noticing these typos. 👍  Even after you mention them, it takes me a while to see them. 😄

A common one that I often mistype is `shcedule` instead of `schedule` (even now, typing it, I almost made the same mistake).